Remove names from product specific conventions

cbudz opened the pull request:
Removes:

- RH associate contacts

Adds:

- Directions for questions on product naming

- Directions for questions about product docs to contact prod docs specific emails where available.

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/product_conventions/amq7.adoc:
```diff
@@ -1,7 +1,6 @@
 
-
-Responsible person: Ben Hardesty (bhardest@redhat.com)
+For information on official product naming, including long- and short-form names, please contact naming@redhat.com.
```
Since using "please" is against IBM style guide, would you mind removing all instances in this PR?

bergerhoffer replied:
Also, the listinfo for naming@redhat.com [1] says that it's an internal list. Just checking whether or not this is something that should be shared publicly.

[1] http://post-office.corp.redhat.com/mailman/listinfo/naming

cbudz replied:
Hey @bergerhoffer 

Azure - I do believe we should evaluate this entire section and define our guidance on third party products. Les was working on something related to this so I'll check with him on status. 

Naming - let me check in with them. I'm of the opinion we shed naming concerns entirely.

leswilliams44 replied:
@cbudz  and @bergerhoffer , I think it's a good idea to define how we refer to all the major cloud providers. For instance, we start off with Microsoft Azure, but we drop the Microsoft in succeeding mentions unless Microsoft Azure is a UI label or some other circumstance that requires the company name in front of the product.

We spell out Amazon Web Services at the beginning of a title or article, but we use AWS almost exclusively.

We really haven't done enough with Google Cloud Platform (GCP) or IBM Cloud to have developed inconsistent citations.

bergerhoffer replied:
Thanks @leswilliams44. What do you think about at least adding some sort of caveat at the top? Something like how this is Red Hat's guidance on referring to the Azure products, but for the latest, see the official Azure documentation, or something?

leswilliams44 replied:
@bergerhoffer, that sounds reasonable. I'll take a stab at it.

bergerhoffer commented:
@leswilliams44 Here is the PR I was talking about, if you want to weigh in with your thoughts on how to handle the Azure / third-party naming guidance going forward.

cbudz commented:
@bergerhoffer - updates to this PR:

- Spoke with brand management who said to use the brand@ mailing list for external resources
- Removed item about naming contact from each file. 
- Updated info in overview doc re: naming contact. 
- All pleases are removed.

bergerhoffer commented:
Thanks @cbudz, this LGTM now. I'm used to always requiring PRs to be squashed to 1 commit. Do you think we should require that with this repo, or be a little looser with that?

cbudz commented:
@bergerhoffer IMO the lower the barrier to entry the better for contributions. I can see that expectation as reasonable for upstream project communities, but ideally I think keeping our contributor guidelines simple and streamlined makes our project more approachable.

bergerhoffer commented:
That works for me. Merging!
